internal/handlers/commonparams/params.go

```diff
@@ -70,17 +70,19 @@ func GetWholeNumberParam(value any) (int64, error) {
 	}
 }
 
-// getWholeParamStrict validates the given value for find and count commands.
+// GetValidatedNumberParamWithBoundary converts and validates a value into a int64 number.
```

Comment:
I think we should also document the errors this function returns.

Reply:
Added.